cmd/secret/main.go

```diff
@@ -1,14 +1,16 @@
 package main
 
 import (
+	"context"
 	"fmt"
-)
 
-var (
-	revision = "unknown"
+	"github.com/go-itools-internship/go-secret/cmd/secret/cmd"
 )
 
 func main() {
-	fmt.Printf("secret %s\n", revision)
-	fmt.Println("Hi from go-secret!")
+	p := cmd.NewRoot()
```

Comment:
I would suggest to simplify constructor name for the sake of readability. Try just `New`. WDYT?